pkgs/development/tools/poetry2nix/poetry2nix/overrides/build-systems.json

```diff
@@ -7208,9 +7208,6 @@
   "igraph": [
     "setuptools"
   ],
-  "ihatemoney": [
```

Comment:
What does this have to do with `python3.pkgs.ihatemoney`?

Reply:
my bad. Fix in https://github.com/NixOS/nixpkgs/pull/238391